src/NuProj.Tasks/NuProj.Tasks.csproj

```diff
@@ -41,8 +41,8 @@
     <AssemblyOriginatorKeyFile>..\Key.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Build.Framework" />
-    <Reference Include="Microsoft.Build.Utilities.v4.0" />
+    <Reference Include="Microsoft.Build.Framework, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
+    <Reference Include="Microsoft.Build.Utilities.v12.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
```

Comment:
No, it wouldn't work without MSBuild 12 installed. But as NuProj itself supports Dev12, it seems impossible to compile something without the thing you're supporting being on the box.
If MSBuild were on NuGet, we could get it from there.